front/app/containers/Admin/projects/edit/participationContext.tsx

```diff
@@ -734,22 +760,45 @@ class ParticipationContext extends PureComponent<
               <>
                 <SectionField>
                   <SubSectionTitle>
-                    <FormattedMessage
-                      {...messages.amountPerCitizen}
-                      values={{ currency: tenantCurrency }}
-                    />
+                    <FormattedMessage {...messages.totalBudget} />
                   </SubSectionTitle>
                   <BudgetingAmountInput
-                    onChange={this.handleBudgetingAmountChange}
+                    onChange={this.handleMinBudgetingAmountChange}
                     type="number"
-                    min="1"
-                    placeholder=""
-                    value={max_budget ? max_budget.toString() : null}
+                    min="0"
+                    value={minBudgetInputValue}
+                    label={
+                      <LabelWrapper>
+                        <FormattedMessage {...messages.minimum} />
+                        <IconTooltip
+                          content={
+                            <FormattedMessage {...messages.minimumTooltip} />
+                          }
+                        />
+                      </LabelWrapper>
+                    }
                   />
-                  <Error
-                    text={noBudgetingAmount}
-                    apiErrors={apiErrors && apiErrors.max_budget}
+                  <Error text={minBudgetError} />
+                  <Error apiErrors={apiErrors && apiErrors.min_budget} />
+                </SectionField>
+                <SectionField>
+                  <BudgetingAmountInput
```

Comment:
With these two `BudgetingAmountInput`s, I think there is a bit of unnecessary repetition- for example, the `type="number"` and what is being passed to the `label` prop. Could be DRYed up

Reply:
True, although it's only 2 components so far and a couple of lines that would be saved. Could certainly be made a separate component, but not going to work on that now. I'm putting this PR in my retrospective of the epic so I'll log it afterwards and we take this up at some point.